Manual.md

```diff
@@ -602,6 +602,11 @@ i.e `register_shell="/bin/tcsh /bin/csh"`
 package metadata and can be queried with `xbps-query` by users.
 i.e for qutebrowser `tags="browser chromium-based qt5 python3"`
 
+- `perl_configure_dirs` A white space separate list of directories defined by relative
+path to `wrksrc` that have Makefile.PL files that need to be processes for the package
+to work. It is used in the perl-module build_style and has no use outside of it.
```

Comment:
I would shorten it a bit:

A white space separated list of directories relative to `wrksrc` containing `Makefile.PL` files which also need to be processed for the package to work.

Reply:
shortened a bit